packages/dds/tree/src/feature-libraries/forest-summary/summaryTypes.ts

```diff
@@ -13,19 +19,25 @@ import type { MinimumVersionForCollab } from "@fluidframework/runtime-definition
 export const forestSummaryKey = "Forest";
 
 /**
- * The key for the blob under ForestSummarizer's root.
- * This blob contains the ForestCodec's output.
- * See {@link ForestIncrementalSummaryBuilder} for details on the summary structure.
+ * The entire contents of the ForestSummarizer's summary (ForestCodec's output) is added to a blob with this key.
+ * This was added in {@link ForestSummaryFormatVersion.v1 | version 1} and is used in
+ * {@link ForestSummaryFormatVersion.v1 | version 1} and {@link ForestSummaryFormatVersion.v2 | version 2}.
  */
-export const forestSummaryContentKey = "ForestTree";
+export const summaryContentBlobKeyV1 = "ForestTree";
 
 /**
+ * From {@link ForestSummaryFormatVersion.v3 | version 3} onwards, the inline portion of the top-level forest content
+ * is stored in a summary blob with this key.
+ * If the summary is not incremental, the content stored is the entire forest content.
+ * If the summary is incremental, the contents of the incremental chunks is stored separately:
  * The contents of an incremental chunk is under a summary tree node with its {@link ChunkReferenceId} as the key.
- * The inline portion of the chunk content is encoded with the forest codec is stored in a blob with this key.
- * The rest of the chunk contents  is stored in the summary tree under the summary tree node.
+ * The inline portion of the chunk content is encoded with the forest codec and is stored in a blob with this key as
+ * well. The rest of the chunk contents  is stored in the summary tree under the summary tree node.
+ *
+ * @remarks
  * See the summary format in {@link ForestIncrementalSummaryBuilder} for more details.
  */
-export const chunkContentsBlobKey = "contents";
+export const summaryContentBlobKeyV3 = "contents";
```

Comment:
I find the name and use of summaryContentBlobKeyV1 and summaryContentBlobKeyV3 a bit confusing. The docs do good job of clarifying, but the constant names are a bit off from how they are used. For example v2 using summaryContentBlobKeyV1 and incremental blob decoding using summaryContentBlobKeyV3 are both a bit strange.

Maybe a rename would help. For example summaryContentBlobKeyV1 -> summaryContentBlobKeyV1V2.
I can't think of a better name for summaryContentBlobKeyV3 that's not super long

Reply:
Yeah, this is confusing. I updated the pattern to match how other places (such as MessageFormat) handles it. Basically, have a file for V1ToV2 and another one for V3. Then when new versions are added, the files can be renamed or new files be added.